CONTRIBUTING.md

```diff
@@ -539,9 +539,22 @@ commit merging your branch with the base branch, so if there is a
 conflict and this merge cannot be performed automatically, the bot
 will put a `needs: rebase` label, and the tests won't run.
 
-Otherwise, a large suite of tests will be run on GitLab, plus some
+Otherwise, a suite of tests will be run on GitLab, plus some
 additional tests on GitHub Actions for Windows and macOS compatibility.
 
+The complete suite of tests is no longer run by default to save
+resources. But it is still required before merging a PR, so this is why
+the bot will put a `needs: full CI` label if it has only run the
+lightweight tests. If you are a member of **@coq/contributors**, you can
+request a full run of the CI by putting the `request: full CI` label
+before pushing to your PR branch, or by commenting
+`@coqbot: run full CI` after having pushed. (In case you need to re-run
```

Comment:
Let's keep the colons consistent. I think not having them is better.

Reply:
This is about the `@coqbot` commands.

Reply:
OK. I'll remove them.